pkgs/development/tools/avro-tools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, makeWrapper, jre, lib }:
+
+stdenv.mkDerivation rec {
+  version = "1.9.0";
+  name = "avro-tools-${version}";
+
+  src = fetchurl {
+    url = "https://apache.claz.org/avro/avro-${version}/java/avro-tools-${version}.jar";
```

Comment:
Can you pull this from maven central?!

Reply:
Yessiree. Want me to use the `fetchMaven` function?

Reply:
I have no issue with `fetchurl`, either way should be fine.

Reply:
Resolved! Stayed with fetchurl for minimum friction.